CONTRIBUTING.md

```diff
@@ -0,0 +1,139 @@
+# Contributing to api-designer
+
+Do you want to help improving this project?? Here are some instructions to get you started. They are probably not perfect, please let us know if anything feels wrong or incomplete.
+
+## Reporting Issues
+
+When reporting [issues](https://github.com/mulesoft/api-designer/issues)
+on GitHub please include the version of api-designer you are using, as well as the name and version of your browser, and your OS. Please also include the steps required to reproduce the problem if possible and applicable. This information will help us review and fix your issue faster.
```

Comment:
Should this be on the same line. Also, an idea, but maybe we could provide a template to fill in? LGTM anyway, probably a little verbose but doesn't hurt :smile: